### ISSUE TITLE
Page needs improvement

### ISSUE THREAD

**Issue by rtipnis** [knative/docs#2426]
No reasoning is given for any of those minikube start parameters. Why 6 cpus, why a vm is needed (and not a container etc.)

**Comment by jonatasbaldin** [comment]
Hi @rtipnis! Could point out which page is this content located?

Thanks!

**Comment by abrennan89** [comment]
I don't think that this issue is valid anymore, since vendor specific install docs have been removed and we shouldn't have references to minikube now.

@rtipnis please let me know if you think that this issue still requires doc changes, and provide links or specific locations of the issues, otherwise I will close it. Thanks.

cc @mattmoor for a second pair of eyes :eyes:
